libraries/botbuilder-dialogs-adaptive/src/actions/deleteProperties.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * @module botbuilder-dialogs-adaptive
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { Dialog, DialogContext, DialogTurnResult, DialogConfiguration } from 'botbuilder-dialogs';
+import { SequenceContext } from '../sequenceContext';
+
+export interface DeletePropertiesConfiguration extends DialogConfiguration {
+    properties?: string[];
+}
+
+export class DeleteProperties<O extends object = {}> extends Dialog<O> {
+
+    public static declarativeType = 'Microsoft.DeleteProperties';
+
+    public constructor();
+    public constructor(properties?: string[]) {
+        super();
+        if (properties) { this.properties = properties; }
+    }
+
+    /**
+     * Collection of property paths to remove.
+     */
+    public properties: string[] = [];
+
+    public configure(config: DeletePropertiesConfiguration): this {
+        return super.configure(config);
+    }
+
+    public async beginDialog(dc: DialogContext, options?: O): Promise<DialogTurnResult> {
+        if (dc instanceof SequenceContext) {
```

Comment:
>if (dc instanceof SequenceContext) { [](start = 8, length = 36)

There's no reason to restrict this to just AdaptiveDialogs anymore.  We moved the state manager down to the base DialogContext so now you should be able to use this action anywhere.

Reply:
I'll remove this condition.